types/handlebars/index.d.ts

```diff
@@ -96,8 +96,8 @@ interface HandlebarsTemplatable {
     template: HandlebarsTemplateDelegate;
 }
 
-interface HandlebarsTemplateDelegate {
-    (context: any, options?: any): string;
+interface HandlebarsTemplateDelegate<T = any> {
+    (context: T, options?: RuntimeOptions): string;
```

Comment:
At first I thought this had to be `T extends object`, but if the template only uses `{{.}}` it can literally be anything.

It could be valid to omit the context too, but I'm not sure that's useful.